README.md

```diff
@@ -1 +1,162 @@
-# hello-microvers
\ No newline at end of file
+# hello-microvers
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```

Comment:
- Kindly remove the micro verse logo to make your project look professional :recycle: